extensions/2.0/Vendor/NV_materials_mdl/schema/functionCall.type.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "$id": "functionCall.type.schema.json",
+    "title": "Type",
+    "type": "object",
+    "description": "MDL type describing either a built-in or user-defined type, or an array of a built-in or user-defined type.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "module": {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
+            "description": "The ID of the containing module.  This field **MUST NOT** be defined if a built-in type is specified."
+        },
+        "typeName": {
+            "type": "string",
```

Comment:
Are empty strings allowed here?

Reply:
Good catch, I fixed it here and for `functionCall.functionName`.